.github/workflows/main.yml

```
@@ -24,9 +24,9 @@ jobs:
       - name: Set env
         id: vars
         run: |
-          echo ::set-output name=tag::${GITHUB_REF#refs/*/}
-          echo ::set-output name=short_commit::$(echo $GITHUB_SHA | cut -c1-7)
-          echo ::set-output name=date::$(date -u +"%Y-%m-%dT%H:%M:%SZ")
+          echo name=${GITHUB_REF#refs/*/} >> $GITHUB_OUTPUT
+          echo name=$(echo $GITHUB_SHA | cut -c1-7) >> $GITHUB_OUTPUT
+          echo name=$(date -u +"%Y-%m-%dT%H:%M:%SZ") >> $GITHUB_OUTPUT
```

mmm, I think we should preserve the name:
```suggestion
          echo tag=${GITHUB_REF#refs/*/} >> $GITHUB_OUTPUT
          echo short_commit=$(echo $GITHUB_SHA | cut -c1-7) >> $GITHUB_OUTPUT
          echo date=$(date -u +"%Y-%m-%dT%H:%M:%SZ") >> $GITHUB_OUTPUT
```